examples/with-jest-typescript/package.json

```diff
@@ -17,7 +17,7 @@
     "@types/next": "^2.4.8",
     "@types/react": "^16.0.41",
     "@types/react-dom": "^16.0.4",
-    "@zeit/next-typescript": "^0.0.11",
+    "@zeit/next-typescript": "1.0.1",
```

Comment:
In this case you have to add the babel preset to `.babelrc`, since it wasn't there before `1.0.0`

Reply:
Should be addressed in the latest commit. Let me know.